MatrixKit/Views/ReadReceipts/MXKReadReceiptTableViewCell.xib

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you please adjust the cell height by using for example 74 (instead of 180)?
For the design of this cell, you should copy the existing ContactTableViewCell.xib (riot-ios) by considering only the following items:`thumbnailView, contactDisplayNameLabel and contactInformationLabel` (Do not rename your properties, your naming is fine). 
You could not define a specific label with a fix string like the "Read" label. We won't be able to translate it. You will have to use an attributed string for the `receiptDescriptionLabel`
(Note: remove the white space before ':': "Read: xxx").